ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndex.scala

```diff
@@ -30,7 +30,7 @@ object JdbcIndex {
         new LedgerBackedIndexService(MeteredReadOnlyLedger(ledger, metrics), participantId) {
           override def getLedgerConfiguration(): Source[v2.LedgerConfiguration, NotUsed] =
             // FIXME(JM): The indexer should on start set the default configuration.
-            Source.single(v2.LedgerConfiguration(timeModel.minTtl, timeModel.maxTtl))
+            Source.single(v2.LedgerConfiguration(initialConfig.maxDeduplicationTime))
```

Comment:
We should probably implement this method properly soon?

Reply:
Yes indeed, I'm going to do that.